<launch>
    <node name="robot_0" pkg="mrs_project2_simulation" type="agent_node.py" output="screen">
        <param name="formation_positions_param" type="string" value="empty_triangle_random_form"/>    
    </node>
    <node name="robot_1" pkg="mrs_project2_simulation" type="agent_node.py" output="screen">
        <param name="formation_positions_param" type="string" value="empty_triangle_random_form"/>    
    </node>
    <node name="robot_2" pkg="mrs_project2_simulation" type="agent_node.py" output="screen">
        <param name="formation_positions_param" type="string" value="empty_triangle_random_form"/>    
        <param name="is_leader" type="bool" value="True"/>    
    </node>
    
    <node name="calc_neighbours_node" pkg="mrs_project2_simulation" type="calc_neighbours_node.py">
        <param name="num_robots" type="int" value="3"/>    
        <param name="adjacency_params" type="string" value="empty_triangle_random_leader_adj"/>    
    </node>


    <node name="stage_node" pkg="sphero_stage" type="start.py">
        <param name="param_file" type="string" value="launch_params_empty_triangle_random"/>
    </node>
</launch>